metal/resource_metal_connection.go

```diff
@@ -53,6 +55,12 @@ func resourceMetalConnection() *schema.Resource {
 				Description: "Connection type - dedicated or shared",
 				ForceNew:    true,
 			},
+			"mode": {
```

Comment:
You could add a validator:

```
ValidateFunc: validation.StringInSlice([]string{"standard", "tunnel"}, false),
```

If you don't mind, also add it to `type` and `redundancy`